### ISSUE TITLE
class declaration is causing `use of undeclared variable: ClipboardJS

### ISSUE THREAD

**Issue by shoulders** [quick-lint/quick-lint-js#1244]
Class declarations are being flagged as undefined variables. I have included one example below

```javascript
// This enables the copy to clipboard buttons
new ClipboardJS('.copyToClipboardBtn', {
    text: function(trigger){
        return trigger.nextElementSibling.innerText;
    }
});
```
![Image](https://github.com/user-attachments/assets/1f6ab9d1-95c4-49c2-b7aa-7af04efb9d1e)

![Image](https://github.com/user-attachments/assets/b0ac8231-1e10-4e77-b3e3-24fbba09a5a3)

### Information
-  v3.2.0
- VSCode v1.101.1
